scripts/build-canister.sh

```diff
@@ -19,10 +19,10 @@ fi
 
 if [[ -z "$FEATURES" ]]; then
   # No features provided
-  cargo build --bin "$CANISTER" --target "$TARGET" --release
+  cargo build -p "$CANISTER" --target "$TARGET" --release
```

Comment:
Maybe leave a comment here explaining why the `-p` option which I guess is non-standard when you want to build a binary.